### PR TITLE
Add SOCIS features documentation

### LINKED ISSUE #333: Update docs to incorporate recent additions to symbolic module!
🐞 **Problem**
The title is self-explanatory.

🎯 **Goal**

  - Add changelog for version `0.2.1` related to changes & additions in symbolic module in `docs/source/changelog.rst`.
  - Update "Symbolic Calculations" section in `docs/source/user_guide.rst` with a little more examples.
  - Update `README.rst` description. It's been a long time!!

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by OVSofia**
Fixes #333

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,24 @@
 What's new
 ==========
 
+einsteinpy 0.2.1 - 2020-06-15
+-----------------------------
+
+This release will bring a dedicated module for the symbolic calculations, which adds some new computations.
```
Comment (ritzvik) [inline docs/source/changelog.rst]: This minor release would bring improvements and new feature additions to the already existing symbolic calculations module.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+
+Part of this release is sponsored by European Space Agency, through Summer of Code in Space
+(SOCIS) 2019 program.
+
+Features
+........
+
+* Symbolic Manipulation of Ricci Scalar, Weyl Tensor, Stress-Energy-Momentum Tensor, Einstein Tensor and Schouten Tensor
```
Comment (ritzvik) [inline docs/source/changelog.rst]:
```
* New tensors in symbolic module

  * Ricci Scalar
  * Weyl Tensor
  * Stress-Energy-Momentum Tensor
  * Einstein Tensor
  * Schouten Tensor

* Improvement in performance of current tensors
```

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+* Tensors can represent arrays of numpy like functions
```
Comment (ritzvik) [inline docs/source/changelog.rst]: `* Lambdify option for tensors`

File: docs/source/changelog.rst [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+* Support for vectors at arbitrary space-time symbolically in Tensor
```
Comment (ritzvik) [inline docs/source/changelog.rst]:
```
* Support for vectors at arbitrary space-time symbolically as 1\ :sup:`st` order tensor.
```

File: docs/source/changelog.rst [hunk 5]
```diff
@@ -1,6 +1,24 @@
...
+* Support for scalars at arbitrary space-time symbolically in Tensor
```
Comment (ritzvik) [inline docs/source/changelog.rst]:
```
* Support for scalars at arbitrary space-time symbolically as 0\ :sup:`th` order tensor.
```

File: docs/source/changelog.rst [hunk 6]
```diff
@@ -1,6 +1,24 @@
 What's new
 ==========
 
+einsteinpy 0.2.1 - 2020-06-15
```
Comment (ritzvik) [inline docs/source/changelog.rst]: Change date to `2019-09-15`. That's when we would probably release.

File: docs/source/changelog.rst (hunk 5 above)
Comment (ritzvik) [reply docs/source/changelog.rst]: Add another line.
```
* Addition of constants sub-module to symbolic module
```

File: docs/source/user_guide.rst [hunk 7]
```diff
@@ -192,7 +187,6 @@ Using the functions:
 
 Symbolic Calculations
 =====================
-EinsteinPy also supports smbolic calculations in :py:class:`~einsteinpy.utils.christoffel`
```
Comment (ritzvik) [inline docs/source/user_guide.rst]: Don't remove this line, edit this into
```
EinsteinPy also supports symbolic calculations in :py:class:`~einsteinpy.symbolic`
```

Comment (ritzvik) [reply docs/source/user_guide.rst]: Also correct the spelling error of `symbolic` here.

File: docs/source/user_guide.rst [hunk 8]
```diff
@@ -208,9 +202,23 @@ EinsteinPy also supports smbolic calculations in :py:class:`~einsteinpy.utils.ch
...
+        import sympy
+        from einsteinpy.symbolic import SchwarzschildMetric, EinsteinTensor
+
+        m = SchwarzschildMetric()
+        G1 = EinsteinTensor.from_metric(m)
+        print(G1.arr)
+
+    .. code-block:: python
+
+        [[a*c**2*(-a + r)/r**4 + a*c**2*(a - r)/r**4, 0, 0, 0], [0, a/(r**2*(a - r)) + a/(r**2*(-a + r)), 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]
```
Comment (ritzvik) [inline docs/source/user_guide.rst]: Have you calculated it? Is it the actual result?

Comment (OVSofia) [reply docs/source/user_guide.rst]: Yes :)
image

File: docs/source/changelog.rst [hunk 9]
```diff
@@ -1,6 +1,33 @@
...
+  * Stress-Energy-Momentum Tensor
+  * Einstein Tensor
+  * Schouten Tensor
+
+* Improvement in performance of current tensors
+
+  * Lambdify option for tensors
+  * Support for vectors at arbitrary space-time symbolically as 1\ :sup:`st` order tensor.
+  * Support for scalars at arbitrary space-time symbolically as 0\ :sup:`th` order tensor.
+  * Addition of constants sub-module to symbolic module
```
Comment (ritzvik) [inline docs/source/changelog.rst]: The above four points are not intended to be sub-points under `Improvement in performance of current tensors`.
```
* Improvement in performance of current tensors
* Lambdify option for tensors
* Support for vectors at arbitrary space-time symbolically as 1\ :sup:`st` order tensor.
* Support for scalars at arbitrary space-time symbolically as 0\ :sup:`th` order tensor.
* Addition of constants sub-module to symbolic module
```

Comment (ritzvik) [reply docs/source/changelog.rst]: After this, please squash n rebase :)

File: docs/source/changelog.rst [hunk 10]
```diff
@@ -1,6 +1,33 @@
 What's new
 ==========
 
+einsteinpy 0.2.1 - 2019-09-15
+-----------------------------
+
+This minor release would bring improvements and new feature additions to the already existing symbolic calculations module.
+Part of this release is sponsored by European Space Agency, through Summer of Code in Space
```
Comment (ritzvik) [inline docs/source/changelog.rst]: Can you add a newline between 1st and 2nd line?

Comment (OVSofia) [reply docs/source/changelog.rst]: Sure :)

File: docs/source/changelog.rst (hunk 9 above)
Comment (OVSofia) [reply docs/source/changelog.rst]: So, `Improvement in performance of current tensors` make reference only to the issue: use symmetry in Christoffel to decrease computing time?

**Comment by OVSofia** [comment]
Sorry, I forgot to make you co-author!  >.<'

**Comment by ritzvik** [comment]
> Sorry, I forgot to make you co-author! >.<'

It's all right. I didn't make any commits.

**Pull request merged**